src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -1573,6 +1584,17 @@ public void updateEnabledState() {
             getBackAction().setEnabled(false);
             getForwardAction().setEnabled(false);
         }
+
+        boolean saved = false;
+
+        if (tabCount > 0) {
```

Comment:
Can you extract the next two lines to a new method `getSelectedPanel` any maybe replace other occurrences of the same code with this method.